### PR TITLE
Problem 24: new solution in C++ added.

### PR DESCRIPTION
**Pull request by namannimmo10**

### CODE REVIEW

**Comment by namannimmo10** [comment]
@ryukinix Can you approve this?

File: Problem024/C++/solution_1.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int main(){
```
Comment (rafaelcn) [inline Problem024/C++/solution_1.cpp]:
```suggestion
int main() {
```

File: Problem024/C++/solution_1.cpp [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  char numbers[] = "0123456789";
+  for(int i=1;i<1000000;i++){
```
Comment (rafaelcn) [inline Problem024/C++/solution_1.cpp]:
```suggestion
  for(int i=1;i<1000000;i++) {
```

**Comment by rafaelcn** [comment]
@namannimmo10 last changes and I approve this.

**Comment by namannimmo10** [comment]
@rafaelcn what about this?

**Pull request merged**